openpype/hosts/traypublisher/plugins/publish/collect_frame_data_from_asset_entity.py

```diff
@@ -23,22 +23,16 @@ class CollectMissingFrameDataFromAssetEntity(
     def process(self, instance):
         if not self.is_active(instance.data):
             return
-        missing_keys = []
+
+        asset_data = instance.data["assetEntity"]["data"]
+
         for key in (
             "fps",
             "frameStart",
             "frameEnd",
             "handleStart",
             "handleEnd"
         ):
-            if key not in instance.data:
-                missing_keys.append(key)
-        keys_set = []
-        for key in missing_keys:
-            asset_data = instance.data["assetEntity"]["data"]
-            if key in asset_data:
-                instance.data[key] = asset_data[key]
-                keys_set.append(key)
-        if keys_set:
-            self.log.debug(f"Frame range data {keys_set} "
-                           "has been collected from asset entity.")
+            instance.data[key] = asset_data[key]
+            self.log.debug(
+                f"Collected Frame range data '{key}':{asset_data[key]} ")
```

Comment:
This logic will now ALWAYS overwrite the collected frame range data which is NOT what I'd expect as a user if my input footage has actual frames in its filename - unless maybe this particular plugin is disabled by default?

Should maybe the default `active` state for this particular plugin be "false" in settings so that it requires the user to ENABLE it to actually say 'no matter what - just use the frame range of the asset as data'?